notebooks/0101_CEDS-prepare.py

```diff
@@ -91,7 +94,7 @@
 
 # %%
 ceds = pd.concat(
-    read_CEDS(Path(ceds_data_folder) / f"{s}_CEDS_emissions_by_country_sector_v{ceds_release}.csv") for s in species
+    read_CEDS(Path(ceds_data_folder) / f"{s}_CEDS_estimates_by_country_sector_v{ceds_release}.csv") for s in species
```

Comment:
CEDS changed their file name?! Mama mia (breaks backwards compatibility with older versions, but that's fine, maybe adding a comment to say that this name was different in earlier versions isn't a terrible idea, although I guess we have git for that, ignore me)

Reply:
Yeah naming is not stable. 
emissions / estimates
_v_YYYY_MM_DD / _vYYYY_MM_DD

Should we in the future do something smart with versioning through the `class CEDSoption` (currently in 0201), by pulling that out to somewhere else?

Reply:
We could, but I think it's probably more trouble than it's worth as we're not going to do this lots and lots of times (I hope 🙃 )